notesapp/migrations/0002_note.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.2 on 2018-10-22 22:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('notesapp', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Note',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('title', models.CharField(max_length=200)),
+                ('content', models.TextField(blank=True)),
+                ('user_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='notesapp.User')),
```

Comment:
Good use of CASCADE here, just be careful if you do end up using any `Delete` commands